getting_started/First_run.md

```diff
@@ -13,7 +13,7 @@ We recommend starting with a single host first and then moving to multihost.
 Local development is a convenient way to run MaxText on a single host. It doesn't scale to
 multiple hosts.
 
-1. [Create and SSH to the single-host VM of your choice.](https://cloud.google.com/tpu/docs/users-guide-tpu-vm#creating_a_cloud_tpu_vm_with_gcloud) We recommend a `v4-8`.
+1. [Create and SSH to the single-host VM of your choice.](https://cloud.google.com/tpu/docs/users-guide-tpu-vm#creating_a_cloud_tpu_vm_with_gcloud) We recommend a `v5litepod-8`.
```

Comment:
We have done the most extensive testing on v4-8 and our unit tests run on v4-8, so I prefer to keep v4-8 as recommended. However everything should work on a single host v5litepod-8 as well - notably the default config in base.yml uses a small enough batch size on v5litepod so step 4 below runs fine.

How about instead "You can use whatever single host capacity is available to you, such as a v4-8, v5litepod-8 or v5p-8. Our unit tests run against v4-8 but any single host VM should work."

Reply:
Yeah, that's much better I think. The root issue was the (surprising) unavailability of v4-8's, so I'll rephrase that.